modules/gke-cluster-standard/variables.tf

```diff
@@ -19,13 +19,18 @@ variable "backup_configs" {
   type = object({
     enable_backup_agent = optional(bool, false)
     backup_plans = optional(map(object({
+      all_namespaces = optional(bool, true)
+      applications = optional(list(object({
+        namespace = string
+        name      = string
+      })))
       encryption_key                    = optional(string)
       include_secrets                   = optional(bool, true)
       include_volume_data               = optional(bool, true)
       namespaces                        = optional(list(string))
       region                            = string
       schedule                          = string
-      retention_policy_days             = optional(string)
+      retention_policy_days             = optional(string, 1)
```

Comment:
This because with _schedule_ defined you must also set Backup Retain Days.

Reply:
So can we make _retention_policy_days_  not optional?

Reply:
I don't think so. It looks like schedule is actually optional:

![image](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/assets/100541/ac0cb2f8-d479-49f2-903a-8b73f5435f34)

Reply:
while you're at it, can you pute required attributes before optional ones?

Reply:
OK @juliocc , so let's make _schedule_ optional too?

Reply:
sgtm!